theme/styles/snippets/product-images-slider.scss

```diff
@@ -10,9 +10,15 @@
   }
 
   .splide__arrow--override {
-    position: absolute;
-    z-index: 1;
-    opacity: 0;
+    //Hide the arrows on mobile, so the images are swipeable.
+    display: none;
+
```

Comment:
The "arrows" were covering the images so the Slider.js drag feature wasn't working. On smaller screens I set the arrows to `display: none` so the images can be swiped.